tests/test_watcher.py

```
@@ -119,3 +119,32 @@ def test_process_new_job_deduplication(watcher_instance):
     assert mock_state.total_new_entries_found == 2
+
+
+def test_seen_jobs_lru_bounds_set_size(watcher_instance):
+    """Ensure the internal seen-jobs tracking does not grow unbounded."""
+    w = watcher_instance
+    w.show_notification = MagicMock()
+    w.state.total_new_entries_found = 0
+    w.state.seen_job_ids = collections.deque(maxlen=50)
+
+    # Pretend config exposes a small max size for the LRU
+    def fake_get(section, key, **kwargs):
```

Ruff (0.14.7)

Unused function argument: `kwargs`

(ARG001)